src/abstractdataframe/prettytables.jl

```diff
@@ -0,0 +1,62 @@
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+#
+# Description
+# ==============================================================================
+#
+#     This file contains the definitions related to the interface with
+#     PrettyTables.jl
+#
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+# Default DataFrames highlighter for text backend.
+#
+# This highlighter changes the text color to gray in cells with `nothing`,
+# `missing`, `#undef`, and types related to DataFrames.jl.
+function _df_h_f(data,i,j)
```

Comment:
can you please use some more informative name (longer name is OK)

Reply:
ah - and please use spaces here and below after `,`.

Reply:
Yes, and sorry about the space in `,`. I am really used to not using spaces there :D